lib/rubocop/target_ruby.rb

```diff
@@ -7,7 +7,7 @@ class TargetRuby
     DEFAULT_VERSION = KNOWN_RUBIES.first
 
     OBSOLETE_RUBIES = {
-      1.9 => '0.50', 2.0 => '0.50', 2.1 => '0.58', 2.2 => '0.69', 2.3 => '0.82'
+      1.9 => '0.50', 2.0 => '0.50', 2.1 => '0.58', 2.2 => '0.69', 2.3 => '0.81'
```

Comment:
Thank you for the explanation. It looks like a misleading variable name. The local variable name `last_version` looks like it should be `version_removed` correctly.